src/test_application/CameraController.cpp

```diff
@@ -0,0 +1,421 @@
+/**
+ * Open Space Program
+ * Copyright © 2019-2020 Open Space Program Project
+ *
+ * MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+#include "CameraController.h"
+
+#include <adera/Machines/UserControl.h>
+
+#include <osp/Satellites/SatVehicle.h>
+
+#include <osp/Active/ActiveScene.h>
+#include <osp/Active/SysAreaAssociate.h>
+#include <osp/Active/SysPhysics.h>
+#include <osp/Active/SysVehicle.h>
+
+using testapp::SysCameraController;
+using testapp::ACompCameraController;
+
+using adera::active::machines::MachineUserControl;
+
+using osp::universe::Satellite;
+using osp::universe::Universe;
+using osp::universe::UCompTransformTraj;
+using osp::universe::UCompVehicle;
```

Comment:
Are all of these using statements really better than just namespacing things as they're used?

Reply:
The code is much harder to follow if there's `osp::universe` and `osp::active` written everywhere. A few of these are only used in certain functions, I think they can be moved down into just the function scopes where they're used.